fastly/backend_test.go

```diff
@@ -63,6 +62,9 @@ func TestClient_Backends(t *testing.T) {
 	if b.OverrideHost != "origin.example.com" {
 		t.Errorf("bad override_host: %q", b.OverrideHost)
 	}
+	if b.SSLCheckCert != true {
```

Comment:
ℹ️ I also confirm the API defaults to setting the `ssl_check_cert` field to `true`.